web/src/app/browser/src/keymanEngine.ts

```diff
@@ -576,8 +633,12 @@ export default class KeymanEngine extends KeymanEngineBase<BrowserConfiguration,
    *  @param  {string=}         argFormFactor   layout form factor, defaulting to 'desktop'
    *  @param  {(string|number)=}  argLayerId    name or index of layer to show, defaulting to 'default'
    *  @return {Object}                          DIV object with filled keyboard layer content
+   *
+   * See https://help.keyman.com/developer/engine/web/17.0/reference/osk/BuildVisualKeyboard
+   *
+   * (May be moved to https://help.keyman.com/developer/engine/web/17.0/reference/core/BuildVisualKeyboard)
```

Comment:
```suggestion
   * See https://help.keyman.com/developer/engine/web/current-version/reference/osk/BuildVisualKeyboard
```

If/when we move it, then we can add the new link.